site/src/xServices/setup/setupXService.ts

```diff
@@ -0,0 +1,108 @@
+import * as API from "api/api"
+import {
+  ApiError,
+  FieldErrors,
+  getErrorMessage,
+  hasApiFieldErrors,
+  isApiError,
+  mapApiErrorToFieldErrors,
+} from "api/errors"
+import * as TypesGen from "api/typesGenerated"
+import { assign, createMachine } from "xstate"
+
+export const Language = {
+  createFirstUserError: "Error on creating the user.",
```

Comment:
```suggestion
  createFirstUserError: Failed to create the user.",
```

I've seen the pattern of "Error on"... frequently in our product, but I think "Failed to" is more human.